packages/amplify-e2e-tests/src/__tests__/cdk/base-cdk.test.ts

```diff
@@ -21,65 +21,67 @@ describe('CDK GraphQL Transformer', () => {
     deleteProjectDir(projRoot);
   });
 
-  test('CDK base case', async () => {
-    const templatePath = path.resolve(path.join(__dirname, 'backends', 'base-cdk'));
-    const name = await initCDKProject(projRoot, templatePath);
-    const outputs = await cdkDeploy(projRoot, '--all');
-    const { GraphQLAPIEndpointOutput: apiEndpoint, GraphQLAPIKeyOutput: apiKey } = outputs[name];
+  ['2.80.0', 'latest'].forEach((cdkVersion) => {
```

Comment:
Is there a reason why we are testing it on CDK 2.80 in addition to the latest?

Reply:
Yeah, this makes sure that our min/max peerDep versions both work.